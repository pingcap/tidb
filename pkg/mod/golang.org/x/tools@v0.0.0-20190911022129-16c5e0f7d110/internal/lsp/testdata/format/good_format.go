package format //@format("package")

import (
	"log"
)

func goodbye() {
	log.Printf("byeeeee")
}
