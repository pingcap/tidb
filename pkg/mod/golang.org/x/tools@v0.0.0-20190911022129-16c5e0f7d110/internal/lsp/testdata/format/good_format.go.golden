-- gofmt --
package format //@format("package")

import (
	"log"
)

func goodbye() {
	log.Printf("byeeeee")
}

-- gofmt-d --

